pkg/curatedpackages/bundlesource_test.go

```diff
@@ -0,0 +1,91 @@
+package curatedpackages_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+)
+
+func TestBundleSourceSet(t *testing.T) {
+	h := newBSHelper(t)
+	h.Parallel()
+
+	type acceptCase struct {
+		// name of the test
+		name string
+		// input from the user
+		input string
+		// expected result
+		expected curatedpackages.BundleSource
+	}
+
+	accepts := []acceptCase{
+		{"golden path", "cluster", curatedpackages.Cluster},
+		{"case insensitivity", "cLuSTEr", curatedpackages.Cluster},
+		{"golden path", "registry", curatedpackages.Registry},
+		{"case insensitivity #2", "Registry", curatedpackages.Registry},
+		{"whitespace before", " registry", curatedpackages.Registry},
+		{"whitespace after", "registry ", curatedpackages.Registry},
+	}
+	for _, testcase := range accepts {
+		h.runAcceptCase(testcase.name, testcase.input, testcase.expected)
+	}
+
+	type rejectCase struct {
+		// name of the test
+		name string
+		// input from the user
+		input string
+	}
+
+	rejects := []rejectCase{
```

Comment:
Same here, move to a table test. And I would probably put it in a different test
So we have one test for the accepts and another for the rejects